File: 30-Days-of-Code/Python/day02_operators.py
#!/bin/python3

import sys

if __name__ == "__main__":
    meal_cost = float(input().strip())
    tip_percent = int(input().strip())
    tax_percent = int(input().strip())

    tip_percent = float(tip_percent/100)
    tax_percent = float(tax_percent/100)
    tip = float(meal_cost * tip_percent)

    tax = float(meal_cost * tax_percent)
    total_cost = round(meal_cost + tip + tax)

    print("The total meal cost is {} dollars.".format(total_cost))

File: 30-Days-of-Code/Scala/day02_operators.scala
object Solution {

    def main(args: Array[String]) {
        val sc = new java.util.Scanner (System.in);
        var meal_cost = sc.nextDouble();
        var tip_percent = sc.nextInt();
        var tax_percent = sc.nextInt();
        println("The total meal cost is " + totalCost(meal_cost, tip_percent, tax_percent) + " dollars.")
    }

    def totalCost(mealCost: Double, tipPercent: Int, taxPercent: Int): Int = {
        val tip = mealCost * (tipPercent.toDouble / 100)
        val tax = mealCost * (taxPercent.toDouble / 100)
        Math.round(mealCost + tip + tax).toInt
    }
}
